test/jdk/java/foreign/TestVarHandleCombinators.java

```diff
@@ -176,16 +107,13 @@ public void testNestedSequenceAccess() {
         //[10 : [5 : [x32 i32]]]
 
         VarHandle vh = MemoryHandles.varHandle(int.class, ByteOrder.nativeOrder());
-        vh = MemoryHandles.withOffset(vh, 4);
-        VarHandle inner_vh = MemoryHandles.withStride(vh, 8);
-        VarHandle outer_vh = MemoryHandles.withStride(inner_vh, 5 * 8);
         int count = 0;
         try (MemorySegment segment = MemorySegment.allocateNative(inner_size * outer_size * 8)) {
             for (long i = 0; i < outer_size; i++) {
                 for (long j = 0; j < inner_size; j++) {
-                    outer_vh.set(segment.address(), i, j, count);
+                    vh.set(segment, i * 40 + j * 8, count);
```

Comment:
Can this be done with combinators instead of manual offsets like before? Mostly curious about how this use-case changes.

Reply:
I can change this, for sure, but in real world, such use case would just go through the layout ABI, which does all the combination you want for free. Are you sure you want me to write what MemoryLayout::varHandle does inside the test?

Reply:
Ok, just wanted to know if it should use the layout API or it couldn't for some reason.